### DISCUSSION TITLE
Best practice to relocate mastodon to another disk

### DISCUSSION THREAD

**Discussion by solick** [mastodon/mastodon#26001] [General]
Hi, i have my mastodon instance running in a Proxmox VM. Now the original disk space seems to be not enough anymore.

I was thinking to create a new disk for the VM and either moving the complete mastodon user or the live folder to this new disk.

I did a first trial but was not able to get mastodon running again. Maybe someone has done this already successfully and could provide some help how to do this?

Thans in advance!

**Comment by StrangeMinds1960** [comment]
I've not tested this but it may help you

https://docs.joinmastodon.org/admin/migrating/
